librz/core/rtr_http.c

```diff
@@ -127,14 +126,20 @@ static int rz_core_rtr_http_run(RzCore *core, int launch, int browse, const char
 		so.accept_timeout = 1;
 	}
 
-	origcfg = core->config;
-	newcfg = rz_config_clone(core->config);
-	core->config = newcfg;
+	// store current configs
+	RzConfigHold *hc = rz_config_hold_new(core->config);
```

Comment:
add a check in case `hc` is NULL.

Reply:
> btw, i like the changes. good job :) There are just some minor details missing.

Thanks a lot 😄. Here are the requested changes.